packages/webapp/src/elections/components/ongoing-election.tsx

```diff
@@ -49,28 +57,93 @@ export const OngoingElection = ({ election }: { election: any }) => {
                 <Text>In progress until 6:30pm EDT</Text>
             </Container>
             <Ongoing.SupportSegment />
-            {/* TODO: How do we get previous round info for rounds that didn't come to consensus? Do that here. */}
-            {roundIndex > 0 &&
-                [...Array(roundIndex)].map((_, i) => (
-                    <Ongoing.CompletedRoundSegment
-                        key={`election-round-${i + 1}`}
-                        roundIndex={i}
-                    />
-                ))}
-            {election?.electionState === ElectionStatus.Final ? (
-                <Ongoing.ChiefsRoundSegment roundEndTime={roundEndTime} />
-            ) : (
-                <Ongoing.OngoingRoundSegment
-                    electionState={election.electionState}
-                    roundIndex={roundIndex}
-                    roundStartTime={roundStartTime}
-                    roundEndTime={roundEndTime}
-                    roundDurationMs={roundDurationMs}
-                    electionConfig={election.config}
-                />
-            )}
+            <CompletedRounds roundIndex={roundIndex} />
+            <SignInContainer />
+            <SignUpContainer />
+            <CurrentRound
+                electionState={election.electionState}
+                roundIndex={roundIndex}
+                roundStartTime={roundStartTime}
+                roundEndTime={roundEndTime}
+                roundDurationMs={roundDurationMs}
+                electionConfig={election.config}
+            />
```

Comment:
Not sure how well git will merge these changes with mine. We *might* want to merge mine first.